packit_service/service/events.py

```diff
@@ -75,6 +78,16 @@ class TestingFarmResult(str, enum.Enum):
     running = "running"
 
 
+class TheJobTriggerType(str, enum.Enum):
```

Comment:
that's a funny name

Reply:
I will probably revert it back but want to be sure I go through all of the occurrences and decide which enum needs to be there.

Reply:
oki

Reply:
Are you ok with merging this as is and revert after I test it properly on stage?

Reply:
I am, that's why I approved